libs/feature-flags/src/lib/configcat.ts

```diff
@@ -14,17 +14,19 @@ export class Client implements FeatureFlagClient {
     const ccConfig: IJSAutoPollOptions = {
       dataGovernance: DataGovernance.EuOnly,
     }
-    if (typeof window === 'undefined') {
-      this.configcat = require('configcat-node').createClient(
-        resolvedSdkKey,
-        ccConfig,
-      )
-    } else {
-      this.configcat = require('configcat-js').createClient(
-        resolvedSdkKey,
-        ccConfig,
-      )
-    }
+    // TODO find a way to conditionally require this without
```

Comment:
Hmm I'm pretty sure I tried this for both webpack and server side, this was what @eirikurn came up for me. IMO we should figure this out now.

Reply:
The errors that come up are:
```
WARNING in /home/dabbeg/repos/island.is/node_modules/got/source/request-as-event-emitter.js 72:18-25
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

WARNING in /home/dabbeg/repos/island.is/node_modules/keyv/src/index.js 18:14-40
Critical dependency: the request of a dependency is an expression

ERROR in /home/dabbeg/repos/island.is/node_modules/got/source/utils/get-body-size.js
Module not found: Error: Can't resolve 'fs' in '/home/dabbeg/repos/island.is/node_modules/got/source/utils'

ERROR in /home/dabbeg/repos/island.is/node_modules/got/source/utils/timed-out.js
Module not found: Error: Can't resolve 'net' in '/home/dabbeg/repos/island.is/node_modules/got/source/utils'

ERROR in /home/dabbeg/repos/island.is/node_modules/tunnel/lib/tunnel.js
Module not found: Error: Can't resolve 'net' in '/home/dabbeg/repos/island.is/node_modules/tunnel/lib'

ERROR in /home/dabbeg/repos/island.is/node_modules/defer-to-connect/dist/index.js
Module not found: Error: Can't resolve 'tls' in '/home/dabbeg/repos/island.is/node_modules/defer-to-connect/dist'

ERROR in /home/dabbeg/repos/island.is/node_modules/tunnel/lib/tunnel.js
Module not found: Error: Can't resolve 'tls' in '/home/dabbeg/repos/island.is/node_modules/tunnel/lib'
ℹ ｢wdm｣: Failed to compile.
```

I was able to supress the errors by adding fs, net, tls to the webpack node config but that was not a very reliable long term solution. Maybe @eirikurn knows how to fix this?